webpack/sass/main.scss

```diff
@@ -30,6 +33,19 @@ $project-card-theme: $blue;
   }
 }
 
+.breadcrumb-container {
+  border-top: 4px solid $color-forest-green;
+  background-color: $color-lighter-gray;
+
+  .breadcrumb {
+    @extend .padding-bigger;
+  
+    .link {
+      @extend .has-color-forest-green;
+    }
+  }
```

Comment:
You shouldn't have to edit styles here. If the component does not match the styles, make the change in Vocabulary itself.

Reply:
This is link color and padding. And I guess that will be different for every website.